test/irb/test_completion.rb

```diff
@@ -107,11 +107,56 @@ def test_complete_require_relative
     end
 
     def test_complete_variable
+      # Bug fix issues https://github.com/ruby/irb/issues/368
+      private_methods = ''
+      methods = ''
+      global_variables = ''
+      local_variables = ''
+      instance_variables = ''
       str_example = ''
-      str_example.clear # suppress "assigned but unused variable" warning
+      @str_example = ''
```

Comment:
It looks like `str_example` and `@str_example` are test subjects, and the rest are for making sure `irb` won't confuse them when completion?

If that's the case, can we add comments to mention that?

Reply:
Thanks comment.
I added comment https://github.com/ruby/irb/pull/369/commits/697708da7ce8ac803dd54dcea5ac7fe23f1a306a.